rplugin/python3/deoplete/sources/deoplete_jedi/worker.py

```diff
@@ -62,16 +62,12 @@ def completion_work(self, cache_key, extra_modules, source, line, col,
     def run(self):
         while True:
             try:
-                work = self.in_queue.get(block=False, timeout=0.5)
+                work = self.in_queue.get()
                 self.log.debug('Got work')
-                self.out_queue.put(self.completion_work(*work), block=False)
+                self.out_queue.put(self.completion_work(*work))
```

Comment:
I stand corrected.  I guess it couldn't hurt to put the timeout back in, just in case.